LS_2.py

```diff
@@ -0,0 +1,27 @@
+from flask import Flask, render_template, request, redirect, make_response
+
+app = Flask(__name__)
+
+@app.route('/')
+def index():
+    return render_template('index.html')
+
+@app.route('/welcome', methods=['POST'])
+def welcome():
+    name = request.form['name']
+    email = request.form['email']
+
+    response = make_response(render_template('welcome.html', name=name))
+    response.set_cookie('name', name)
+    response.set_cookie('email', email)
+    return response

```

Comment:
Можно эти функции объединить
```suggestion
@app.route('/', methods=['GET', 'POST'])
def index():
    if request.method == 'POST':
        name = request.form['name']
        email = request.form['email']

        response = make_response(render_template('welcome.html', name=name))
        response.set_cookie('name', name)
        response.set_cookie('email', email)
        return response

    return render_template('index.html')
```